src/binder/bind/bind_graph_pattern.cpp

```diff
@@ -334,6 +334,9 @@ std::pair<uint64_t, uint64_t> Binder::bindVariableLengthRelBound(
         throw BinderException(
             "Lower bound of rel " + relPattern.getVariableName() + " is greater than upperBound.");
     }
+    if (relPattern.getRelType() == QueryRelType::ALL_SHORTEST && lowerBound != 1) {
```

Comment:
Also for QueryRelType::SHORTEST

And change or documentation (in Match clause) accordingly.